packages/botfuel-dialog/src/nlus/botfuel-trainer-nlu.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2017 - present, Botfuel (https://www.botfuel.io).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require('fs');
+const validUrl = require('valid-url');
+const rp = require('request-promise-native');
+const dir = require('node-dir');
+const logger = require('logtown')('BotfuelTrainerNlu');
+const BooleanExtractor = require('../extractors/boolean-extractor');
+const LocationExtractor = require('../extractors/location-extractor');
+const CompositeExtractor = require('../extractors/composite-extractor');
+const SdkError = require('../errors/sdk-error');
+const Intent = require('./intent');
+const Nlu = require('./nlu');
+
+/**
+ * NLU using Botfuel Trainer API
+ */
+class BotfuelTrainerNlu extends Nlu {
+  /** @inheritdoc */
+  constructor(config) {
+    logger.debug('constructor', config);
+    super(config);
+    this.extractor = null;
+    this.qna = null;
+
+    if (!process.env.BOTFUEL_APP_TOKEN) {
+      throw new SdkError('BOTFUEL_APP_TOKEN are required for using the nlu service');
+    }
+
+    if (!validUrl.isUri(config.trainerApiUri)) {
```

Comment:
Is this temporary?
Config should not define trainerApiUri ...

Reply:
Ok agree. I propose process.env.BOTFUEL_TRAINER_API_URL || "https://...." . What do you think?